src/components/AppMap.vue

```diff
@@ -65,8 +66,89 @@
           <code>bonus</code>: Minimum bonus modifier tier (0, 1, 2 or 3)<br>
           <code>region</code>: Tower region name (e.g. Tabantha, Lanayru, Hateno) <br>
           <code>fieldarea</code>: Field Map area (0 - 93)<br>
-          <code>lotm</code>: Spawns with Lord of the Mountain (1 or 0)<br>
-        </p>
+          <code>lotm</code>: Spawns with Lord of the Mountain (1 or 0) <br>
+          <code>location</code>: items possibly in, beneath or above something
+
+          <details class="ml-3">
+            <summary>Locations</summary>
+            <ul class="small">
+              <li>Castle Tower</li>
+              <li>Corridor</li>
+              <li>Cryonis Gate</li>
+              <li>Dining Hall</li>
+              <li>Docks</li>
+              <li>East Hallway</li>
+              <li>East Passage</li>
+              <li>First Gatehouse</li>
+              <li>Guards' Chambers</li>
+              <li>Guards' Chambers Annex</li>
+              <li>Kings Study</li>
+              <li>Library</li>
+              <li>Library Corridor</li>
+              <li>Lockup</li>
+              <li>Lockup Upper</li>
+              <li>Observation Room</li>
+              <li>Observation Room Balcony</li>
+              <li>Sanctum</li>
+              <li>Second Gatehouse</li>
+              <li>Stairs</li>
+              <li>Watch Tower</li>
+              <li>West Main Corridor</li>
+              <li>West Passage</li>
+              <li>Zelda's Room</li>
+              <li>Zelda's Study</li>
+            </ul>
+          </details>
+
+          <code>korok_type</code>: types of koroks<br>
```

Comment:
nit:

```suggestion
          <code>korok_type</code>: Korok puzzle type<br>
```

(to be consistent with the other descriptions)